Tests/WebUITests/WebViewProxyTests.swift

```diff
@@ -87,3 +210,67 @@ struct WebViewProxyTests {
         #expect(oldInstance != newInstance)
     }
 }
+
+private func wait<V: Equatable & Sendable>(
+    for sequence: AsyncPublisher<Published<V?>.Publisher>,
+    condition: @escaping @Sendable (V?) -> Bool,
+    timeout: Duration
+) async throws -> Bool {
+    try await withThrowingTaskGroup(of: Bool.self) { group in
+        defer { group.cancelAll() }
+
+        group.addTask {
+            try await Task { @MainActor in
+                for try await value in sequence {
+                    if condition(value) {
+                        return true
+                    }
+                }
+                return false
+            }
+            .value
+        }
+
+        group.addTask {
+            try await Task.sleep(for: timeout)
+            return false
+        }
+
+        guard let result = try await group.next() else {
+            return false
+        }
+        return result
+    }
+}
+
+private func wait<V: Equatable & Sendable>(
+    for sequence: AsyncPublisher<Published<V>.Publisher>,
+    condition: @escaping @Sendable (V) -> Bool,
+    timeout: Duration
+) async throws -> Bool {
```

Comment:
I felt that the following interface is more in line with Swift’s style.
What are your thoughts?
```suggestion
private func waitForValue<V: Equatable & Sendable>(
    in sequence: AsyncPublisher<Published<V>.Publisher>,
    equalsTo expectedValue: V,
    timeout: Duration
) async throws -> Bool {
```

Reply:
I was initially going to use `toEqual` instead of `condition`, but I chose condition because I wanted to use `equalTo()` for `CGSize` and `CGPoint` comparisons.

However, it seems that there is no particular difference between `==` and `equalTo()` in practice.
So, I think it would be better to go with something that is more natural for Swift.

Reply:
Changed in [b573db9](https://github.com/cybozu/WebUI/pull/59/commits/b573db930887fdd02465869f95936856d6f5da85)